### PR TITLE
Created New ManagedBatchParser project in .NetStandard

### PR DESCRIPTION
**Pull request by dhruvanmurthy**
Creating a new PR, same as #762.

### CODE REVIEW

File: src/Microsoft.SqlTools.ManagedBatchParser/BatchParser/TextBlock.cs [hunk 1]
```diff
@@ -8,17 +8,17 @@
...
         private readonly Parser parser;
         private readonly IEnumerable<Token> tokens;
 
         /// <summary>
         /// Constructor for the TextBlock class 
         /// </summary>
-        public TextBlock(Parser parser, Token token) : this(parser, new[] { token })
-        {
-        }
+        //public TextBlock(Parser parser, Token token) : this(parser, new[] { token })
```
Comment (kburtram) [inline src/Microsoft.SqlTools.ManagedBatchParser/BatchParser/TextBlock.cs]: should this code be deleted instead of commented out?  also, it is a public method that presumably could be used by consumers of the library.  Is there any particular reason to remove this constructor?

File: src/Microsoft.SqlTools.ManagedBatchParser/Properties/AssemblyInfo.cs [hunk 2]
```diff
@@ -0,0 +1,38 @@
+﻿using System.Reflection;
```
Comment (kburtram) [inline src/Microsoft.SqlTools.ManagedBatchParser/Properties/AssemblyInfo.cs]: this file should have a copyright header similar to the other .cs files in this project.

File: src/Microsoft.SqlTools.ManagedBatchParser/Properties/AssemblyInfo.cs [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+
+// General Information about an assembly is controlled through the following 
+// set of attributes. Change these attribute values to modify the information
+// associated with an assembly.
+[assembly: AssemblyTitle("Microsoft.SqlTools.ManagedBatchParser.Properties")]
```
Comment (kburtram) [inline src/Microsoft.SqlTools.ManagedBatchParser/Properties/AssemblyInfo.cs]: not sure this is a good assembly title..same note on assembly product.

File: src/Microsoft.SqlTools.ManagedBatchParser/ReliableConnection/ReliableConnectionHelper.cs [hunk 4]
```diff
@@ -705,6 +705,13 @@ public class ServerInfo
             public Dictionary<string, object> Options { get; set; }
         }
 
+        public class ClusterEndpoint
```
Comment (kburtram) [inline src/Microsoft.SqlTools.ManagedBatchParser/ReliableConnection/ReliableConnectionHelper.cs]: seems like an unrelated change?

File: src/Microsoft.SqlTools.ManagedBatchParser/BatchParser/TextBlock.cs (hunk 1 above)
Comment (dhruvanmurthy) [reply src/Microsoft.SqlTools.ManagedBatchParser/BatchParser/TextBlock.cs]: Reverted it

File: src/Microsoft.SqlTools.ManagedBatchParser/Properties/AssemblyInfo.cs (hunk 2 above)
Comment (dhruvanmurthy) [reply src/Microsoft.SqlTools.ManagedBatchParser/Properties/AssemblyInfo.cs]: Added header

**Comment by kburtram** [comment]
@dhruvanmurthy Please confirm how this change impacts code coverage prior to check-in.  The latest code coverage results I've seen showed that ~5000 executable statements are no longer covered with this change relative to the baseline.

**Comment by kburtram** [comment]
@dhruvanmurthy @NiranjanVirtuosity I merged this PR with a few minor changes in https://github.com/Microsoft/sqltoolsservice/pull/774.  I think the code coverage will be down since the new assembly won't be included, so we can follow-up on any remaining issues as subsequent PRs.
